CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project are documented in this file.
 
 ## [0.2.5] - 2022-07-06
 
+ - `Button`: Add a `checkable` property that turns the button into a toggle
+   button. Use the new `checked` property to query whether the toggle button
+   is pressed down or not.
+
```

Comment:
This is in the wrong section!

Reply:
Good catch! Sorry for that!

I did the change several times since auto-format changed the entire file more often than I care to admit. Looks like I need to work on my tolerance to frustration levels rising ;-)